components/registry-facade/pkg/blobserve/blobserve.go

```diff
@@ -32,19 +31,17 @@ type Server struct {
 }
 
 // Config configures a server.
-// Mapping maps a single path prefix to an image name, e.g.
-//    { "theia": "eu.gcr.io/typefox/gitpod/theia-ide:" } maps an incoming request from
-//    /theia/foobar.1/index.html to a file named index.html in the first layer blob of
-//    the eu.gcr.io/typefox/gitpod/theia-ide:foobar.1 image.
 type Config struct {
 	Port    int           `json:"port"`
 	Timeout util.Duration `json:"timeout,omitempty"`
-	Mapping map[string]struct {
-		Repository string `json:"repo"`
-		Workdir    string `json:"workdir,omitempty"`
-	} `json:"mapping"`
-	Preparation []string `json:"preparation,omitempty"`
-	BlobSpace   struct {
+	Repos   map[string]struct {
```

Comment:
This field is not specifically related to a repository, right? It's more the image string prefix, no matter whether it's a repository or image name?

Reply:
It refers to [Docker image repositories](https://docs.docker.com/docker-hub/repos/) ... it's unfortunate that they've choosen that name becuase it invites confusion like this one :/